docs/content/reference/data_types/point2d.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Point2D
+order: 7
+---
+`Point2d` represents a singular point in two-dimensional space with optional color, radii, and label. `Point2d` entities will be drawn as part of the 2D Spatial SpaceView.
+
+It is compatible with [`AnnotationContext`](../../concepts/annotation-context.md). `class_id` can be used to provide
```

Comment:
that's true for most things actually, not sure if we want to repeat that text on all of them? 🤔 
Sadly not all ofc..